hudi-common/src/main/java/org/apache/hudi/common/table/timeline/HoodieArchivedTimeline.java

```diff
@@ -49,34 +60,76 @@
  * This class can be serialized and de-serialized and on de-serialization the FileSystem is re-initialized.
  */
 public class HoodieArchivedTimeline extends HoodieDefaultTimeline {
+  private static final Pattern ARCHIVE_FILE_PATTERN =
+          Pattern.compile("^\\.commits_\\.archive\\.([0-9]*)$");
 
   private static final String HOODIE_COMMIT_ARCHIVE_LOG_FILE = "commits";
```

Comment:
This is just the prefix right ? May be rename is HOODIE_COMMIT_ARCHIVE_LOG_FILE_NAME_PREFIX ?

Reply:
regex captures everything [0-9]* at the end gives the suffix and is used for sorting files by 'version'

Reply:
I'm referring to this variable `HOODIE_COMMIT_ARCHIVE_LOG_FILE` that stores just the string `commits`, how is that related to the regex ?

Reply:
oh, my bad. I didnt change that line, so i thought you were referring to the line i changed. I can change HOODIE_COMMIT_ARCHIVE_LOG_FILE to include its prefix